src/Libraries/Microsoft.Extensions.ObjectPool.DependencyInjection/ObjectPoolServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,94 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.ObjectPool;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Extensions.DependencyInjection;
```

Comment:
Should this be Microsoft.Extensions.ObjectPool?

Reply:
I believe all extension methods for `ServiceCollection` are in this namespace (at least in aspnetcore) such that you don't need to add a using statement to get it in intellisense. By your comment I assume it's not the case in dotnet/extensions, intentionally?

Reply:
It isn't. We use one namespace for feature area usually.